### ISSUE TITLE
Live demo?

### ISSUE THREAD

**Issue by dandv** [arsnebula/supertags#7]
Would be nice to have one. Trying the Quick Start errored out because `getTags()` wasn't a method of `sTags`.

**Comment by skifaster** [comment]
I'll see what I can do over the weekend

**Comment by Pushplaybang** [comment]
Yeah this would be great, was looking into building this myself, but if you've already got what looks like a full system would be great to either use or fork this.
